foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/HelmChartsNameMigration.java

```diff
@@ -0,0 +1,80 @@
+package com.boozallen.aissemble.upgrade.migration;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.util.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import static com.boozallen.aissemble.upgrade.util.FileUtils.getRegExCaptureGroups;
+
+public class HelmChartsNameMigration extends AbstractAissembleMigration {
+    public static final Logger logger = LoggerFactory.getLogger(HelmChartsNameMigration.class);
+    private static final String dependencyNameReplaceRegex = "(aissemble-[^\\/\\n:]+)(?<!-chart)$";
+    private static final String appNameReplaceRegex = "(^\\s*)(aissemble-[^\\/\\n:]+)(?<!-chart):$";
+    private static final String COLON = ":";
+    private static final String SECOND_REGEX_GROUPING = "$2";
+    private static final String CHART_POSTFIX = "-chart";
+
+
+    @Override
+    protected boolean shouldExecuteOnFile(File file) {
+        boolean shouldExecute = false;
+        try {
+            if (file.getName().matches("Chart.yaml")){
+                ArrayList<String> dependencyName = getRegExCaptureGroups(dependencyNameReplaceRegex, file);
+                shouldExecute = !dependencyName.isEmpty();
```

Comment:
A: also super minor, but sometimes using the following can be more clear to read:
```suggestion
                shouldExecute = CollectionUtils.isNotEmpty(dependencyName);
```